From fcf03a814b8706b305b0c8a6fbadc3eb1f505b9d Mon Sep 17 00:00:00 2001
From: Mohsen Dolaty <mohsen.dolaty@amd.com>
Date: Fri, 7 Jan 2022 11:40:07 -0600
Subject: [PATCH 1/1] phosphor-post-code-manager: Del POST Index from LCD

Delete POST Code Index from LCD display
There is no need for it

Signed-off-by: Mohsen Dolaty <mohsen.dolaty@amd.com>
---
 src/post_code.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/post_code.cpp b/src/post_code.cpp
index 0807393..e8a6918 100644
--- a/src/post_code.cpp
+++ b/src/post_code.cpp
@@ -50,7 +50,7 @@ void writeLCD(uint16_t pcIndex)
     if (pf != NULL)
     {
         if (fgets(pc, POST_CODE_LEN , pf) != NULL) {
-            sprintf(post, "POSTCODE(%2d)%llx   ", pcIndex, atoll(pc));
+            sprintf(post, "POSTCODE: %llx  ", atoll(pc));
             pclose(pf);
             if (lcdlib_open_dev() == 0 ) {
                 lcdlib_write_string(POST_CODE, (unsigned char*)post, LCD_MAX_CHAR);
--
2.25.1
